README.md

```diff
@@ -79,6 +79,7 @@ Flags:
       "subs": [
         "template"
       ],
+      "type": "git",
```

Comment:
Hmm. Sorry, I missed this when talking in #3, but on second thought, this shouldn't be required in this example because the `github.com` host on the next line causes the `type` to be set to `git` automatically. It happens here:

https://github.com/leighmcculloch/vangen/blob/f8cfe81ae25d2ec8a80c00ccdf9cef9e2f41569f/generate_package.go#L51-L52

Is there a case where this isn't happening?

Or maybe you're using a non-GitHub host and we could do with an example that shows how to use non-GitHub hosts with it?

Reply:
I'm using a self hosted GitLab instance.  I didn't realize the minimal example was GitHub specific.  A short note indicating that's the case would be helpful.  I updated my pull request to include a short note indicating the `type` and `source` properties are added automatically for GitHub projects.